pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.depproj

```diff
@@ -0,0 +1,125 @@
+    <PropertyGroup Condition="'@(_runtimeCLR)' != ''">
+      <_runtimePackageId>%(_runtimeCLR.NuGetPackageId)</_runtimePackageId>
+      <_runtimePackageVersion>%(_runtimeCLR.NuGetPackageVersion)</_runtimePackageVersion>
+      <_runtimePackagePath>$(PackagesDir)/$(_runtimePackageId)/$(_runtimePackageVersion)</_runtimePackagePath>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(NuGetRuntimeIdentifier)' != ''">
+      <!-- RID-specific: include all runtime files -->
+      <_FilesToPackage Include="@(ReferenceCopyLocalPaths)">
+        <!-- ResolveNugetPackageAssets doesn't preserve the asset type (native), 
+             calculate it by looking for native in the path -->
+        <IsNative Condition="$([System.String]::new('%(Identity)').ToLowerInvariant().Replace('\', '/').Contains('/native/'))">true</IsNative>
+      </_FilesToPackage>
+      <_FilesToPackage>
+        <TargetPath Condition="'%(_FilesToPackage.IsNative)' != 'true'">runtimes/$(NuGetRuntimeIdentifier)/lib/$(PackageTargetFramework)</TargetPath>
+        <TargetPath Condition="'%(_FilesToPackage.IsNative)' == 'true'">runtimes/$(NuGetRuntimeIdentifier)/native</TargetPath>
+      </_FilesToPackage>
+      <_FilesToPackage Include="$(_runtimePackagePath)/tools/*.*">
+        <NuGetPackageId>$(_runtimePackageId)</NuGetPackageId>
+        <TargetPath>tools</TargetPath>
+      </_FilesToPackage>
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(NuGetRuntimeIdentifier)' == ''">
+      <!-- Not RID-specific: include all reference files -->
+      <_FilesToPackage Include="@(Reference)">
+        <TargetPath>ref/$(PackageTargetFramework)</TargetPath>
+      </_FilesToPackage>
+    </ItemGroup>
+
+    <ItemGroup>
+      <!-- add metadata for source and package -->
+      <_FilesToPackage>
+        <SourcePath>%(Identity)</SourcePath>
+        <PackagePath>%(TargetPath)/%(FileName)%(Extension)</PackagePath>
+      </_FilesToPackage>
+
+      <!-- transform to package path for deduping -->
+      <_primaryFilesByPackagePath Include="@(_FilesToPackage->'%(PackagePath)')" Condition="$([System.String]::new('%(NuGetPackageId)').EndsWith('$(PrimaryPackage)'))"/>
+      <_secondaryFilesByPackagePath Include="@(_FilesToPackage->'%(PackagePath)')" Condition="!$([System.String]::new('%(NuGetPackageId)').EndsWith('$(PrimaryPackage)'))"/>
+
+      <!-- remove any secondary files with same package path as primary files -->
+      <_secondaryFilesByPackagePath Remove="@(_primaryFilesByPackagePath)" />
+
+      <_filesByPackagePath Include="@(_primaryFilesByPackagePath);@(_secondaryFilesByPackagePath)" />
+
+      <!-- transform back to source -->
+      <FilesToPackage Include="@(_filesByPackagePath->'%(SourcePath)')" />
+    </ItemGroup>
+
+    <!-- simple check to make sure we don't accidentally pull an old 
+         package that should have been merged to the primary package -->
+    <ItemGroup>
+      <PermittedSystemPackages Include="System.ValueTuple" />
```

Comment:
I think we can remove this exception (and likely can do so for netcoreapp too).  I believe ValueTuple is being included in flat package now.